core/sample.h

```diff
@@ -21,7 +21,7 @@ struct sample                         // BASE TYPE BYTES  UNITS    RANGE
 	pressure_t pressure[MAX_SENSORS]; // int32_t  2x4    mbar   (0-2 Mbar)             cylinder pressures (main and CCR o2)
 	o2pressure_t setpoint;            // uint16_t   2    mbar   (0-65 bar)             O2 partial pressure (will be setpoint)
 	o2pressure_t o2sensor[MAX_O2_SENSORS];// uint16_t 6x2    mbar   (0-65 bar)             Up to 6 PO2 sensor values (rebreather)
-	bearing_t bearing = { -1 };       // int16_t    2  degrees  (-1 no val, 0-360 deg) compass bearing
+	bearing_t bearing = { .degrees = -1 };// int16_t    2  degrees  (-1 no val, 0-360 deg) compass bearing
```

Comment:
This is an example for where the 'vertical alignment' whitespace pattern comes apart for me - if we'd wanted to stick to the pattern we'd need to add 20+ lines with whitespace only changes to the pull request.